doc/dev/installation.rst

```diff
@@ -207,6 +207,12 @@ To make MLIR libraries discoverable to the compiler:
 
   export MLIR_LIB_DIR="$PWD/mlir/llvm-project/build/lib"
 
+To make Enzyme libraries discoverable to the compiler:
```

Comment:
In the installation guideline, there's no mention of Catalyst's dependency to Enzyme libs. It may be good to add this to the list of installation-from-src requirements.

Reply:
What do you mean? Enzyme is not a prerequisite since we build it from source. Can you point out the specific place you'd like to modify?

Reply:
I couldn't find any proper place to add this to the page. Nevermind, I thought that we're including a list of submodules (dependencies) that we build from source.